### PR TITLE
Jenkins Python Systemd Module Rework - closes #526

### LINKED ISSUE #526: Jenkins Python Systemd Module Rework
The following needs updated in this module:
- prereq install scripts
  - jenkins (sudo) user
  - pythonadm user
  - python3 installed
  - virtualenv
- update the service script
  - remove `EnvironmentFile`
  - change application execution to `python app.py`

### PR DESCRIPTION
**Pull request by bob-crutchley**
Fixing bugs and other issues that came up when delivering this module.

### CODE REVIEW

File: topics/jenkins/modules/python-flask-freestyle-project/README.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+There are a few prerequisites for this module, so make sure the following has been configured on your machine by running the script below:
+- Linux (Ubuntu/Debian) operating system, not in a container such as Docker
+- Jenkins installed & running
+- Python 3 installed
+- `virtualenv` installed
+- `jenkins` user, configured as a sudo user with no password
+- `pythonadm` user created
+
+```bash
+# create the users if they don't exist
```
Comment (LukeBenson) [inline topics/jenkins/modules/python-flask-freestyle-project/README.md]: Ran in a fresh VM with only jenkins installed, and it didn't add pythonadm

File: topics/jenkins/modules/python-flask-freestyle-project/README.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+users=( jenkins pythonadm )
+for user in users; do
+    if ! awk -F : '{ print $1 }' /etc/passwd | grep -w "${user}" > /dev/null; then
+        sudo useradd -m -s /bin/bash ${user}
+    fi
+done
+# add jenkins to sudoers
```
Comment (LukeBenson) [inline topics/jenkins/modules/python-flask-freestyle-project/README.md]: Didn't add jenkins to the sudoers

File: topics/jenkins/modules/python-flask-freestyle-project/README.md [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+- **Setup the python virtual environment and dependencies.**  
+  A virtual environment can be used to keep deployments consistent, dependencies can be stored in `requirements.txt` file and loaded from there.
+- **Start the service.**  
+  The application will need to be stared for it to be functional.
+
+These requirements can be accomplished by putting the script below into an `Execute shell` build step on the Jenkins job:
+
+```bash
+# move to module folder
+cd ./topics/jenkins/modules/basic-python-flask-freestyle-project-deployment-with-systemd/resources
```
Comment (LukeBenson) [inline topics/jenkins/modules/python-flask-freestyle-project/README.md]:
```suggestion
cd ./topics/jenkins/modules/python-flask-freestyle-project/resources
```

File: topics/jenkins/modules/python-flask-freestyle-project/README.md (hunk 1 above)
Comment (bob-crutchley) [reply topics/jenkins/modules/python-flask-freestyle-project/README.md]: Should be working now.

File: topics/jenkins/modules/python-flask-freestyle-project/README.md (hunk 3 above)
Comment (bob-crutchley) [reply topics/jenkins/modules/python-flask-freestyle-project/README.md]: Noice.

**Pull request merged**